Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ansible-core = "==2.12.7"
+ansible-lint = "==6.3.0"
+jmespath = "==1.0.1"
+molecule = "==4.0.1"
+molecule-docker = "==2.0.0"
+molecule-ec2 = "==0.4"
+boto3 = "==1.24.3"
+pyhamcrest = "==2.0.3"
+pylint = "==2.14.5"
+pytest-testinfra = "==6.8.0"
+requests = "==2.28.1"
+yamllint = "==1.27.1"
+dnspython = "==2.2.1"
```

Comment:
move those development packages (they are not required to run ansible itself) to the dev-packages section below

then you need to add `--dev` to the pipenv install command